gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -147,3 +159,79 @@ def expand(self, pcoll):
                       beam.io.BigQueryDisposition.WRITE_APPEND
                       if self._append
                       else beam.io.BigQueryDisposition.WRITE_TRUNCATE))))
+
+  def _update_bigquery_schema(self, update_schema):
+    # type: (bool) -> None
+    if not update_schema or not self._schema:
+      return
+    # if table does not exist, do not need to update the schema.
+    output_table_re_match = re.match(
+        r'^((?P<project>.+):)(?P<dataset>\w+)\.(?P<table>[\w\$]+)$',
+        self._output_table)
+    credentials = GoogleCredentials.get_application_default().create_scoped(
+        ['https://www.googleapis.com/auth/bigquery'])
+    client = bigquery.BigqueryV2(credentials=credentials)
+    try:
+      project_id = output_table_re_match.group('project')
+      dataset_id = output_table_re_match.group('dataset')
+      table_id = output_table_re_match.group('table')
+      found_table = client.tables.Get(bigquery.BigqueryTablesGetRequest(
+          projectId=project_id,
+          datasetId=dataset_id,
+          tableId=table_id))
+    except exceptions.HttpError:
+      return
+
+    new_schema = bigquery.TableSchema()
+    new_schema.fields = _merge_field_schemas(self._schema.fields,
```

Comment:
I think it's actually better to swap the order of the args since when appending, I think you'd expect the order of the columns to remain the same as before and new ones just added to the end rather than the order potentially changing. So, you'd call with `(existing_table.schema.fields, self._schema.fields)`. WDYT?